README.md

```diff
@@ -116,6 +116,39 @@ logger::log_threshold("DEBUG", namespace = "shiny.telemetry")
 
 _note_: This command can be run before the Shiny call or by adding it to the `.Rprofile`.
 
+## Contributing
+_*Have you read the [Contributing Guidelines](https://github.com/Appsilon/.github/blob/main/CONTRIBUTING.md)?*_
+
+### pre-commit
+This project uses [pre-commit][] hooks. The hooks are a series of automated checks run locally.
+These automate mundane tasks such as running spellchecking, linter, and other potential issues
+before you even push your changes!
+
+[pre-commit]: https://pre-commit.com
+
+#### Getting Started
```

Comment:
It's also possible to install `pre-commit` via the R package `{precommit}`

This prevents from running `pre-commit run`, but avoids OS-dependent instructions.

I've installed via this method as `pip` failed on linux and this was the best alternative I've found (or use `pipx`

```R
install.packages(c("precommit", "reticulate"))
precommit::install_precommit()
precommit::use_precommit()
```

Reply:
How about we add another line:

> - using `{precommit}` package: follow [the install steps in its vignette](https://lorenzwalthert.github.io/precommit/articles/precommit.html#installation)

I was reluctant on adding this information, because I have vague memory of `{reticulate}` being quite troublesome and I didn't want introducing another dependency to get the hooks going.

Reply:
Mentioning `{precommit}` as an alternative installation sounds good to me, but with the minimal version (just the reference to documentation).